### ISSUE TITLE
glutTimerFunc can be invoked with a NULL callback

### ISSUE THREAD

**Issue by rcmaniac25** [dcnieho/FreeGLUT#44]
As discussed in email thread from 11/18/2015, subject "[Possible Bugs](http://sourceforge.net/p/freeglut/mailman/message/34549196/)"

"glutTimerFunc has the same issue as #43, but the segfault will happen when the timer is to be invoked. In this case I would think that the timer shouldn't be created if the callback is NULL, as what use would it have?"

**Comment by dcnieho** [comment]
As this appear to be the same issue as #43 and it may well have the same solution as that one, I'm keeping all discussion about the null callbacks in #43 until there is reason to diverge

**Comment by rcmaniac25** [comment]
I kept them separate because #43 _could_ still be used with a NULL callback, but this issue would have no use with a NULL callback.

**Comment by dcnieho** [comment]
Fair enough ;) That may well turn out to be a very good reason to split these issues
